users/src/api/routes/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const usersRouter = require('./users.routes');
+
+const routerApi = (app) => {
+    const router = express.Router();
+    app.use('/api/v1', router);
+    router.use('/users', usersRouter);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ce-box/CE5508-astraoke-api/pullRequest?prid=8478842)
```suggestion
    router.use("/users", usersRouter);
```